Makefile

```diff
@@ -15,6 +15,9 @@
 #  USE_Z3_BINARY=yes
 #      Whether to use the Z3 binary or to build Z3 from source
 #
+#  SYSTEM_LLVM=(no|13, 14, 15, etc...)
```

Comment:
I would like to set this to be 13 by default.
It should build when running `s2e build` outside of docker.